pkg/deploy/api/types.go

```diff
@@ -328,3 +328,22 @@ type DeploymentConfigRollbackSpec struct {
 	// IncludeStrategy specifies whether to include the deployment Strategy.
 	IncludeStrategy bool
 }
+
+// DeploymentLogs represents the logs for a deployment
+type DeploymentLogs struct {
+	kapi.TypeMeta
+	kapi.ListMeta
+}
+
+// DeploymentLogOptions is the REST options for a deployment log
+type DeploymentLogOptions struct {
+	kapi.TypeMeta
+
+	// Follow if true indicates that the deployment log should be streamed until
+	// the deployment terminates.
+	Follow bool
+
+	// NoWait if true causes the call to return immediately even if the deployment
+	// is not available yet. Otherwise the server will wait until the deployment has started.
+	NoWait bool
```

Comment:
How about inverting this to something like `Immediate` or `Wait` which defaults to `false`?